lib/ace/ext/papeeria/latex_behaviour.coffee

```diff
@@ -0,0 +1,187 @@
+define((require, exports, module) ->
+  Behaviour = require("ace/mode/behaviour").Behaviour
+
+  isCommentToken = (token) -> token? and /comment/.test(token.type)
```

Comment:
@khazhoyan well, I'm just extracting common functionality into a function. I think it's justified.
@dbarashev aren't regexps faster?